attractions/time-picker/time-picker.svelte

```diff
@@ -121,29 +129,46 @@
       <div class="shown-on-focus">
         <Button noRipple on:click={() => focus = false}>close the time picker</Button>
       </div>
-      <Label>Hours</Label>
+      <Label>
```

Comment:
But now there's a duplicate `<Label>` element

Reply:
Oh that was :shit:. Fixed. @aabounegm the hooks work fine now--don't need to skip the linting.

Reply:
Honestly, I did development outside of the project and copy-pasted the code (which is why I had the nested Label) because I had the issues with my local prettier changing the code and the broken linting. How are you guys developing the components?

Reply:
Well, I develop in Atom (I don't know if you're aware but Atom's community is falling apart and the editor itself is barely kept alive, let alone feature-rich and up-to-date) so I don't have any sort of Prettier intervening with my code. @aabounegm uses VS code, maybe he can provide some more insight here.

Reply:
I didn't mean the editor, I meant the project itself. Are you developing the components in the same repo locally, or pulling it out into a different sandbox repo for development?

Reply:
I don't quite get the question, but we develop on this repo directly

Reply:
So maybe you are actually testing out components in the docs app as you are developing them? I am just wondering how you render and test the component as you are working on it, without pulling it out of the components folder and putting it in some small app of your own.

Reply:
I personally just quickly created a Snowpack app and put the components there so I could get a better dev environment with hot reloading to see how the component works.

Reply:
Yes. Initially, we had a small draft page where we just played around with the components to test them, but we threw that out once we had proper docs and just used the docs for "testing", and I said before, the docs do not depend on the version published on npm due to using yarn workspaces.
_If curious, you can take a look here: https://github.com/illright/attractions/commit/1b4ac4c3e5ec6a83e11553702a1d81aabd649eea_